vnext/Microsoft.ReactNative.Cxx/ReactPropertyBag.h

```diff
@@ -267,6 +273,14 @@ struct ReactPropertyBag {
         }
       }
 
+      return std::optional<T>{};
+    } else {
+      if (obj) {
+        if (auto temp = obj.try_as<IBoxedValue>()) {
+          return std::optional<T>{BoxedValue<T>::GetValueUnsafe(temp)};
```

Comment:
don't you need a std::move and make `temp` a && ?

Reply:
No, the static GetValueUnsafe method receives const&. It uses to call the GetPtr method. Moving value here would not optimize anything.

---
In reply to: [424138773](https://github.com/microsoft/react-native-windows/pull/4884#discussion_r424138773) [](ancestors = 424138773)